gdal/port/cpl_http.cpp

```diff
@@ -758,27 +758,44 @@ void* CPLHTTPSetOptions(void *pcurl, const char * const* papszOptions)
     }
 #endif
 
-    // Enable following redirections.  Requires libcurl 7.10.1 at least.
+    // Enable following redirections.  Requires libcurl 7.19.1 at least.
+#if LIBCURL_VERSION_NUM >= 0x071301
     curl_easy_setopt(http_handle, CURLOPT_FOLLOWLOCATION, 1 );
     curl_easy_setopt(http_handle, CURLOPT_MAXREDIRS, 10 );
     curl_easy_setopt(http_handle, CURLOPT_POSTREDIR, CURL_REDIR_POST_ALL );
+#endif//LIBCURL_VERSION_NUM >= 0x071301
```

Comment:
Spaces please.  2 before the // and one after.

`#endif  // LIBCURL_VERSION_NUM >= 0x071301`